Makefile

```diff
@@ -191,7 +191,7 @@ TRAVIS_BRANCH?=$(shell git symbolic-ref --short HEAD --quiet)
 # Use git describe to get the latest version tag, commits since then, sha and dirty flag, this
 # results is something like "v1.2.0-13-ab6cedf-dirty"
 NO_TAG ?= "no-tag"
-VERSION := $(shell (git describe --tags --match 'v*.0' --long --dirty || echo $(NO_TAG)) | sed -re 's/(\.0)?-/./')
+VERSION := $(shell (git describe --tags --match 'v*.*.*' --long --dirty || echo $(NO_TAG)) | sed -re 's/(\.0)?-/./')
```

Comment:
What is this fixing?

Reply:
You have got it in commit:
[me@here esp-link]$ git describe --tags --match 'v*.0' --long --dirty | sed -re 's/(.0)?-/./'
v3.1.47-g9c6530d-dirty
[me@here esp-link]$ git describe --tags --match 'v*..' --long --dirty | sed -re 's/(.0)?-/./'
v3.2.47.alpha.0-g9c6530d-dirty

fix needed as it takes wrong version ... into VERSION variable. try it.

Reply:
@tve is it clear now?